crates/payload/builder/src/error.rs

```diff
@@ -27,6 +30,20 @@ pub enum PayloadBuilderError {
     /// Thrown if the payload requests withdrawals before Shanghai activation.
     #[error("withdrawals set before Shanghai activation")]
     WithdrawalsBeforeShanghai,
+    /// Thrown when a transaction fails to convert to a
+    /// [reth_primitives::TransactionSignedEcRecovered].
+    #[cfg(feature = "optimism")]
+    #[error("failed to convert deposit transaction to TransactionSignedEcRecovered")]
+    TransactionEcRecoverFailed,
+    /// Thrown when the L1 block info could not be parsed from the calldata of the
+    /// first transaction supplied in the payload attributes.
+    #[cfg(feature = "optimism")]
+    #[error("failed to parse L1 block info from L1 info tx calldata")]
+    L1BlockInfoParseFailed,
+    /// Thrown when a database account could not be loaded.
+    #[error("failed to load account {0:?}")]
+    #[cfg(feature = "optimism")]
+    AccountLoadFailed(Address),
```

Comment:
same pattern, please move to separate enum and add transparent variant

Reply:
Addressed in https://github.com/anton-rs/op-reth/pull/131